bot/discordbot.py

```diff
@@ -583,7 +583,50 @@ async def test(ctx):
     locked = discord.ui.Button(label='test')
     view.add_item(locked)
 
-    # await message.channel.send(embed=embed, view=view)
+@bot.event
+async def on_member_update(before, after):
+    BACKEND_URL = env["DISCORD_BACKEND_URL"]
+    guild = before.guild
+    user_id = after._user.id
+
+    try:
+        if "humanID-Verified" in [role.name for role in before.roles] and "humanID-Verified" not in [role.name for role in after.roles]:
+            print(f"humanID-Verified role has been removed for user {user_id}")
+            response = requests.delete(
+                BACKEND_URL + '/api/removeUser/?userId={}'.format(user_id)
+            )
+            if response.status_code == 200:
+                print("User is deleted from the database")
+                success = True
+            if success:
+                log_channel = await ensure_text_channel(guild, None,"logs")
+                if not log_channel:
+                    return
+                await log_channel.send(f"humanID-verified role has been removed for user {before.name}")
+    except Exception as e:
+        await log_channel.send(f"Error occurred while deleting {before.name} from database: {str(e)}")
+        
+@bot.event
+async def on_member_remove(member):
+    try:
+        user_id = member.id
+        BACKEND_URL = env["DISCORD_BACKEND_URL"]
+        response = requests.delete(
+            BACKEND_URL + '/api/removeUser/?userId={}'.format(user_id)
+        )
+        if response.status_code == 200:
+            print("User is deleted from the database")
```

Comment:
could log rather than print the status results

Reply:
done